openshift/templates/nsp/deploy.yaml

```diff
@@ -0,0 +1,32 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: global-nsp-build-template
+objects:
+  - kind: NetworkSecurityPolicy
+    apiVersion: security.devops.gov.bc.ca/v1alpha1
+    metadata:
+      name: pods-to-k8s-api
+      labels:
+        name: pods-to-k8s-api
+        env: ${ENV_NAME}
+    spec:
+      description: |
+        Allow pods to talk to the internal k8s api so deployments work.
+        This only needs to be specified once per environment.
+      source:
+        - - $namespace=${NAMESPACE}-${ENV_NAME}
+      destination:
+        - - int:network=internal-cluster-api-endpoint
+
+parameters:
+  - name: NAMESPACE
+    displayName: The target namespace for the resources.
+    required: true
+    value:
+  - name: ENV_NAME
```

Comment:
```suggestion
  - name: TAG_NAME
```
By changing to `TAG_NAME` you take advantage of the openshift-developer-tools `genParams.sh` behavior of updating the tag for you to `dev`, `test`, and `prod` in the generated param files.

Reply:
Done, thanks for catching/suggesting this. Please note I have not had a chance to fully test the handling of the NSPs with the new scripts as I was more focused on wrapping up the series of issuers/verifiers first.